index.html

```diff
@@ -1,11 +1,93 @@
 <!DOCTYPE html>
-<html lang="en">
+<html>
   <head>
-    <meta charset="UTF-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link rel="stylesheet" href="style.css" />
-    <title>Document</title>
+    <title>lesson 4</title>
+    <link rel="stylesheet" href="lesson.css" />
```

Comment:
This should be 
```suggestion
    <link rel="stylesheet" href="style.css" />
```

as your css file is named "style.css"